modules/expectations/session_expectations.lua

```diff
@@ -6,13 +18,16 @@ local Event = events.Event
 local SUCCESS = expectations.SUCCESS
```

Comment:
@aderiabin If you do a little refactoring, then remove these 2 variables as redundant